src/main/java/de/tum/in/www1/artemis/service/AssessmentService.java

```diff
@@ -176,7 +178,39 @@ private double calculateTotalScore(Double calculatedScore, Double maxScore) {
      * @param assessments the List of Feedback
      * @return the total score
      */
-    protected Double calculateTotalScore(List<Feedback> assessments) {
-        return assessments.stream().mapToDouble(Feedback::getCredits).sum();
+    public Double calculateTotalScore(List<Feedback> assessments) {
```

Comment:
you changed the method completely :-) please also adapt the JavaDoc comment above.
Some additional inline comments would also help